--- knewstuff-5.79.0/CMakeLists.txt
+++ knewstuff-5.79.0/CMakeLists.txt
@@ -21,7 +21,7 @@
 include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
 
 set(REQUIRED_QT_VERSION 5.14.0)
-find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE REQUIRED COMPONENTS Widgets Xml)
+find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE REQUIRED COMPONENTS Widgets Xml PrintSupport)
 find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE COMPONENTS Qml Quick)
 
 find_package(KF5Archive ${KF5_DEP_VERSION} REQUIRED)
@@ -37,6 +37,9 @@
 find_package(KF5TextWidgets ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5WidgetsAddons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5XmlGui ${KF5_DEP_VERSION} REQUIRED)
+find_package(KF5GuiAddons ${KF5_DEP_VERSION})
+find_package(KF5DBusAddons ${KF5_DEP_VERSION})
+find_package(KF5Crash ${KF5_DEP_VERSION})
 
 include(ECMGenerateExportHeader)
 include(ECMSetupVersion)
