--- kate-20.12.2.bak/CMakeLists.txt
+++ kate-20.12.2/CMakeLists.txt
@@ -37,12 +37,26 @@
 
 # Required here so that the version comparison below works
 find_package(Qt5Widgets ${QT_MIN_VERSION} CONFIG REQUIRED)
+find_package(Qt5PrintSupport)
+find_package(UDev)
 
 # Required here so that the ki18n_install/kdoctools_install calls injected by
 # the release scripts work.
 find_package(KF5 ${KF5_DEP_VERSION}
     REQUIRED COMPONENTS
         I18n
+        DBusAddons
+        Crash
+        IconThemes
+        GuiAddons
+        Attica
+        Archive
+        CoreAddons
+        WindowSystem
+        ItemViews
+        ConfigWidgets
+        WidgetsAddons
+        Package
     OPTIONAL_COMPONENTS
         DocTools
 )
