--- kiconthemes-5.79.0/CMakeLists.txt
+++ kiconthemes-5.79.0/CMakeLists.txt
@@ -48,6 +48,7 @@
 find_package(KF5Archive ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5I18n ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5CoreAddons ${KF_DEP_VERSION} REQUIRED)
+find_package(KF5GuiAddons ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5ConfigWidgets ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5WidgetsAddons ${KF_DEP_VERSION} REQUIRED)
 find_package(KF5ItemViews ${KF_DEP_VERSION} REQUIRED)
--- kiconthemes-5.79.0.bak/src/CMakeLists.txt
+++ kiconthemes-5.79.0/src/CMakeLists.txt
@@ -111,7 +111,7 @@
 ecm_generate_pri_file(BASE_NAME KIconThemes LIB_NAME KF5IconThemes DEPS "widgets" FILENAME_VAR PRI_FILENAME INCLUDE_INSTALL_DIR ${KDE_INSTALL_INCLUDEDIR_KF5}/KIconThemes)
 install(FILES ${PRI_FILENAME} DESTINATION ${ECM_MKSPECS_INSTALL_DIR})
 
-add_library(KIconEnginePlugin MODULE kiconengineplugin.cpp)
+add_library(KIconEnginePlugin kiconengineplugin.cpp)
 
 target_link_libraries(KIconEnginePlugin
     PRIVATE
